### PR TITLE
Added pre-prod folder in content directory. Updated "REACT_APP_ABOUT_CONTENT_URL" to point pre-prod folder.

### PR DESCRIPTION
**Pull request by iksheth**

### CODE REVIEW

File: public/injectEnv.js [hunk 1]
```diff
@@ -4,6 +4,6 @@ window.injectedEnv ={
 	REACT_APP_USER_LOGOUT_URL:'https://k9dc.essential-dev.com/fence/logout',
 	REACT_APP_BACKEND_API: 'https://caninecommons-dev.cancer.gov/v1/graphql/',
 	REACT_APP_APPLICATION_VERSION:'YYYY_MM_DD/HH:MM',
-	REACT_APP_ABOUT_CONTENT_URL:'https://raw.githubusercontent.com/CBIIT/bento-icdc-frontend/master/src/content/dev/aboutPagesContent.yaml',
+	REACT_APP_ABOUT_CONTENT_URL:'https://raw.githubusercontent.com/CBIIT/bento-icdc-frontend/master/src/content/pre-prod/aboutPagesContent.yaml',
```
Comment (AjayDoddapaneni) [inline public/injectEnv.js:7]: In this case, we can remove dev stage and qa folders?

Comment (iksheth) [reply public/injectEnv.js:7]: Currently QA is having build and they are testing, at this point removing Dev and QA might break tat build. I can remove once build is passed from QA.

Comment (AjayDoddapaneni) [reply public/injectEnv.js:7]: Is it still the case?

Comment (iksheth) [reply public/injectEnv.js:7]: Updated PR.

**Review by AjayDoddapaneni — APPROVED** [review]

**Pull request merged**